### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ab28dev**
Added Requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ The shortcuts that can be used:
 ### Downloads
 [Release](https://github.com/antfu/live-draw/releases)
 
+### Requirements
+- Windows OS
+- .Net Core 5 x86/x64
```
Comment (antfu) [inline README.md:32]: Could make it a link to the download page

**Comment by antfu** [comment]
Oh you are fast :)

File: README.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ The shortcuts that can be used:
 ### Downloads
 [Release](https://github.com/antfu/live-draw/releases)
 
+### Requirements
+- Windows OS
```
[the remark this answers is not in the document]
Comment (ab28dev) [reply README.md:31]: I wanted to write it as Windows 7 or later. But I don't have a windows 7 OS to test. I wanted to contact you but I didn't knew how. Can you tell me some fast way to contact you?

[the remark this answers is not in the document]
Comment (antfu) [reply README.md:31]: Maybe just `Windows 10` then

[the remark this answers is not in the document]
Comment (antfu) [reply README.md:31]: Discord antfu#3212

File: README.md (hunk 1 above)
Comment (ab28dev) [reply README.md:32]: nice idea will do that. And from next time I will ask you to merge pull request because I am a newbie on git.

**Comment by ab28dev** [comment]
> Oh you are fast :)

Sorry I didn't know that I need to ask to review before merge. Please forgive me I am trying to learn.

**Comment by antfu** [comment]
No need to apologize, it's total fine for doing that. I might skip issues but I will review PRs, just thought you are making PR to ask me for review. If you don't need me to review (you are confident enough or just small changes), feel free to push commit directly.

**Pull request merged**